packages/gatsby/index.d.ts

```diff
@@ -17,6 +17,8 @@ import { GraphQLOutputType } from "graphql"
 import { PluginOptionsSchemaJoi, ObjectSchema } from "gatsby-plugin-utils"
 import { IncomingMessage, ServerResponse } from "http"
 
+export type AvailableFeatures = "imageService"
```

Comment:
What are your thoughts on creating a `Features` enum (or object) with `ImageService` being a key on it?

Not a huge functional difference, though I, personally, prefer working with enums as opposed to static strings.

Reply:
In this case I would say no because there is no real piont for an ENUM because that would mean we would need to export it from gatsby as well and now types != gatsby package so it's a pain to maintain.

```
enum AvailableFeatures {
  IMAGE_SERVICE: 'image-service'
}
```